e2e/selecting_tenant_fail_test.go

```diff
@@ -77,27 +77,27 @@ var _ = Describe("creating a Namespace without a Tenant selector when user owns
 	It("should fail", func() {
 		ns := NewNamespace("fail-ns")
 		By("user owns 2 tenants", func() {
-			EventuallyCreation(func() error { return k8sClient.Create(context.TODO(), t1)}).Should(Succeed())
-			EventuallyCreation(func() error { return k8sClient.Create(context.TODO(), t2)}).Should(Succeed())
+			EventuallyCreation(func() error { return k8sClient.Create(context.TODO(), t1) }).Should(Succeed())
```

Comment:
Please, remove additional and unrequired spaces in this file, unless for a specific reason.

Reply:
done